modules/eks/ray-on-eks/modulestack.yaml

```diff
@@ -0,0 +1,122 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: This stack deploys a Module specific IAM permissions
+
+Parameters:
+  ProjectName:
+    Type: String
+    Description: The name of the project
+  DeploymentName:
+    Type: String
+    Description: The name of the deployment
+  ModuleName:
+    Type: String
+    Description: The name of the Module
+  RoleName:
+    Type: String
+    Description: The name of the IAM Role
+  EksClusterAdminRoleArn:
+    Type: String
+    Description: Role from the EKS module with cluster admin permissions
+
+Resources:
+  Policy:
+    Type: "AWS::IAM::Policy"
+    Properties:
+      PolicyDocument:
+        Statement:
+          - Action:
+              - "autoscaling:DescribeAutoScalingGroups"
+            Effect: Allow
+            Resource:
+              - "*"
+          - Action:
+              - "sts:GetCallerIdentity"
+            Effect: Allow
+            Resource:
+              - "*"
+          - Action:
+              - "s3:Create*"
+              - "s3:Put*"
+              - "s3:Delete*"
+              - "s3:Get*"
+              - "s3:List*"
+            Effect: Allow
+            Resource:
+              - "arn:aws:s3:::cdk*"
```

Comment:
I dont think you need S3 permissions in the module. Can you confirm?

Reply:
Removed - thanks!